### PR TITLE
DDCE-2078:HMRC Mongo migration

### PR DESCRIPTION
**Pull request by manpreetsingh1000**

### CODE REVIEW

**Comment by arturopala** [comment]
Hi @manpreetsingh1000 , you can also take a look at https://github.com/hmrc/trader-services-route-one-frontend/commit/78751f331cef0825e928fcdb75b7ca4a834ee6f9

**Comment by manpreetsingh1000** [comment]
@arturopala thank you, will have a look.

**Comment by anburoach** [comment]
Fix unused import warnings in views and routes

File: app/uk/gov/hmrc/homeofficesettledstatus/repository/JourneyCacheRepository.scala [hunk 1]
```diff
@@ -16,13 +16,22 @@
...
 
-import javax.inject.Inject
-import play.modules.reactivemongo.ReactiveMongoComponent
-import uk.gov.hmrc.cache.repository.CacheMongoRepository
 import uk.gov.hmrc.homeofficesettledstatus.config.AppConfig
+import uk.gov.hmrc.mongo.{MongoComponent, TimestampSupport}
 
+import javax.inject.{Inject, Singleton}
 import scala.concurrent.ExecutionContext
+import scala.concurrent.duration._
```
Comment (anburoach) [inline app/uk/gov/hmrc/homeofficesettledstatus/repository/JourneyCacheRepository.scala:24]: Remove unused import

File: app/uk/gov/hmrc/homeofficesettledstatus/services/HomeOfficeSettledStatusFrontendJourneyService.scala [hunk 2]
```diff
@@ -18,9 +18,9 @@ package uk.gov.hmrc.homeofficesettledstatus.services
 
 import javax.inject.{Inject, Singleton}
 import play.api.libs.json.Format
-import uk.gov.hmrc.cache.repository.CacheMongoRepository
 import uk.gov.hmrc.crypto.ApplicationCrypto
 import uk.gov.hmrc.homeofficesettledstatus.journeys.{HomeOfficeSettledStatusFrontendJourneyModel, HomeOfficeSettledStatusFrontendJourneyStateFormats}
+import uk.gov.hmrc.homeofficesettledstatus.repository.{CacheRepository, JourneyCacheRepository}
```
Comment (anburoach) [inline app/uk/gov/hmrc/homeofficesettledstatus/services/HomeOfficeSettledStatusFrontendJourneyService.scala:23]: Remove unused import

File: app/uk/gov/hmrc/homeofficesettledstatus/services/MongoDBCachedJourneyService.scala [hunk 3]
```diff
@@ -17,9 +17,9 @@
 package uk.gov.hmrc.homeofficesettledstatus.services
 
 import play.api.libs.json.{Format, Json}
-import uk.gov.hmrc.cache.repository.{CacheMongoRepository, CacheRepository}
 import uk.gov.hmrc.crypto.json.{JsonDecryptor, JsonEncryptor}
 import uk.gov.hmrc.crypto.{ApplicationCrypto, CompositeSymmetricCrypto, Protected}
+import uk.gov.hmrc.homeofficesettledstatus.repository.{CacheRepository}
```
Comment (anburoach) [inline app/uk/gov/hmrc/homeofficesettledstatus/services/MongoDBCachedJourneyService.scala:22]: remove the curly braces

**Comment by anburoach** [comment]
Please do change in config file
https://github.com/hmrc/hmrc-mongo/blob/master/MIGRATION.md#deployment

**Review by anburoach — APPROVED** [review]
Approved

**Pull request merged**